### PR TITLE
Add jwt components to the project

### PR DESCRIPTION
**Pull request by HGChamara**
complete the development

### CODE REVIEW

File: src/main/java/com/putchar/jwtgenvalidate/config/JwtRequestFilter.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+	@Autowired
+	private JwtuserDetailsService jwtuserDetailsService;
+	
+	@Autowired
+	private JwtTokenUtil jwtTokenUtil;
+
+	@Override
+	protected void doFilterInternal(HttpServletRequest request, HttpServletResponse response, FilterChain filterChain)
+			throws ServletException, IOException
+	{
```
Comment (putchar91) [inline src/main/java/com/putchar/jwtgenvalidate/config/JwtRequestFilter.java:35]: add comment

File: src/main/java/com/putchar/jwtgenvalidate/controller/JwtAuthenticationController.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+	@Autowired
+	private AuthenticationManager authenticationManager;
+	
+	@Autowired
+	private JwtTokenUtil jwtTokenUtil;
+	
+	@Autowired
+	private JwtuserDetailsService userDetailsService;
+	
+	@RequestMapping(value="/authenticate", method=RequestMethod.POST)
```
Comment (putchar91) [inline src/main/java/com/putchar/jwtgenvalidate/controller/JwtAuthenticationController.java:33]: add comment

File: src/main/java/com/putchar/jwtgenvalidate/config/JwtRequestFilter.java [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+{
+	@Autowired
+	private JwtuserDetailsService jwtuserDetailsService;
+	
+	@Autowired
+	private JwtTokenUtil jwtTokenUtil;
+
+	@Override
+	protected void doFilterInternal(HttpServletRequest request, HttpServletResponse response, FilterChain filterChain)
+			throws ServletException, IOException
```
Comment (putchar91) [inline src/main/java/com/putchar/jwtgenvalidate/config/JwtRequestFilter.java:34]: add comment

**Review by putchar91 — CHANGES_REQUESTED** [review]
please resolve these issues

File: src/main/java/com/putchar/jwtgenvalidate/controller/JwtAuthenticationController.java (hunk 2 above)
Comment (HGChamara) [reply src/main/java/com/putchar/jwtgenvalidate/controller/JwtAuthenticationController.java:33]: done

**Review by putchar91 — APPROVED** [review]
approve

**Pull request merged**